### ISSUE TITLE
NVDA + MathPlayer doesn't work

### ISSUE THREAD

**Issue by abnermneves** [nvaccess/nvda#8896]
[nvda.log](https://github.com/nvaccess/nvda/files/2532001/nvda.log)

Hello,

I recently found out about MathPlayer because I needed a software to read LaTeX formulas. But I can't get it to work. I followed the tutorials at the MathPlayer site, but even so it doesn't work.

### Steps to reproduce:
1. Open this link with Firefox or IE (I've seen a video in which MathPlayer works on this exact link)
https://en.wikipedia.org/wiki/Pentagon#Derivation_of_the_diagonal_length_formula

2. Place the cursor over one of the formulas

### Actual behavior:
With Firefox, it reads only the number or symbol over which I placed the cursor
With IE, the formulas aren't even compiled, they're shown as code

### Expected behavior:
It should read the entire formula

#### NVDA Installed/portable/running from source:
Installed
I have also MathType installed in the 30 days trial

#### NVDA version:
2018.3.2

#### Windows version:
Windows 10, 64 bit

#### Name and version of other software in use when reproducing the issue:
Firefox 63.0 (64 bit)
IE 11

#### Other information about your system:
It's in Brazil (is that important?)

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
@abnermneves are you still having this issueß Have you also followed the instruction in NVDA's user guide?

**Comment by Adriani90** [comment]
We didn't hear any updates from the initial author upon request since many years. I am closing this as abandoned. Note that MathPlayer has been succeeded by Mathcat or access8math add-ons, which you can find in the NVDA add-on store.
Closing as abandoned. If you are having these issues with mathcat or access8math, please comment and we can reopen.
